riotfile.py

```diff
@@ -1491,6 +1491,9 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         Venv(
             name="botocore",
             command="pytest {cmdargs} tests/contrib/botocore",
+            env={
+                "DD_DATA_STREAMS_ENABLED": "true",
+            },
             venvs=[
                 Venv(pys=select_pys(min_version="3.8"), pkgs={"moto[all]": latest, "botocore": latest}),
```

Comment:
Instead of setting `DD_DATA_STREAMS_ENABLED=true` for all test runs we should run all the botocore tests with both dsm enabled and disabled.

```suggestion
                Venv(pys=select_pys(min_version="3.8"), pkgs={"moto[all]": latest, "botocore": latest}),
                # Test botocore integration with dsm support
                Venv(pys=[MIN_PYTHON_VERSION, MAX_PYTHON_VERSION], pkgs={"moto[all]": latest, "botocore": latest}, env={
                "DD_DATA_STREAMS_ENABLED": ["false", "true"]},),
```